server/src/test/java/org/elasticsearch/index/replication/RecoveryDuringReplicationTests.java

```diff
@@ -250,18 +254,30 @@ public void testRecoveryAfterPrimaryPromotion() throws Exception {
                 equalTo(totalDocs - 1L));
 
             // index some more
-            int moreDocs = shards.indexDocs(randomIntBetween(0, 5));
+            int moreDocs = randomIntBetween(0, 5);
```

Comment:
why did you unpack this?

Reply:
This was needed for the initial commit. I removed it.